canonical/ubuntu.com
# Hide picto on small screens - /server/management

## Done
- Clean up indentation
- Rename misspelt partial and corrected include paths
- Hide "Got a question?" picto on small screens on '/_ubuntu_advantage_community.html'
- Deleted rogue `<p>` tag from list 
## QA
- Navigate to /server/management on small screen
- Check picto next to "Got a question" is not visible
## Issue / Card

Fixes: #337
